coins

```diff
@@ -21,7 +22,8 @@
     "txversion": 4,
     "overwintered": 1,
     "txfee": 1000,
-    "mm2": 1
+    "mm2": 1,
+    "required_confirmations": 4
```

Comment:
Can't be used for KMD as it would disable notarization requirement. It is trivial to double spend KMD with a conf requirement of `4`. Please re-enable dpowconfs.

Reply:
And it certainly makes sense to keep it in place for all PoW ACs

Reply:
it was one confirmation for KMD before, now it is 4, but you can change it
i will use 4, since nota takes 30 minutes
notarization is in place for all ACs that have notarizations enabled

Reply:
changed KMD to require nota